edx_proctoring/api.py

```diff
@@ -1802,6 +1802,19 @@ def _get_proctored_exam_context(exam, attempt, user_id, course_id, is_practice_e
     """
     Common context variables for the Proctored and Practice exams' templates.
     """
+    password_assistance_url = ''
+    try:
+        password_assistance_url = reverse('password_assistance')
+    except NoReverseMatch:
+        log.exception(u"Can't find password reset link")
+
+    scheme = 'https' if getattr(settings, 'HTTPS', 'on') == 'on' else 'http'
+    password_url = '{scheme}://{site_name}{password_assistance_url}'.format(
```

Comment:
Shall we update the `password_url` to be `''` when password_assistant_url is empty? Your logic would print out an invalid URL.